server/asset/btc/btc.go

```diff
@@ -506,7 +555,7 @@ func (btc *Backend) InitTxSizeBase() uint32 {
 
 // FeeRate returns the current optimal fee rate in sat / byte.
 func (btc *Backend) FeeRate(_ context.Context) (uint64, error) {
```

Comment:
Once we get around to updating `(*RPCClient).call` to accept a context we should pass it through `estimateFee`, esp. if we hit the "hard way" path.  In fact, a ctx could still be quite helpful in those loops juts by doing `if ctx.Err() != nil { return ctx.Err() }` in case it's shutdown or canceled so it doesn't keep trying `getrawtransaction`s e.g.